lumberdev/atelier
# Feat/Navigation Announcement Bar Styling

### WHAT IT DOES:

- Adds theme styling to theme
  - (Text in front of a colored section is either black or white, whichever is higher contrast)
- Adds announcement bar
- Adds logo
- Adds header expansion  to handle overflowing corrections (can be changed to something else if preferable) 

### WHAT TO TEST:

- Adds different colors, border radius & logo
- Check it correctly displays on store
- Check text & logo are legible 

### PREVIEW:

![image](https://github.com/lumberdev/atelier/assets/57941781/439e157e-a9ef-487e-87b5-7b37597c73d1)
![image](https://github.com/lumberdev/atelier/assets/57941781/7c0de3de-a798-4d13-b179-ebac430a884a)
